lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +142,23 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+          options[:color_mode] = :automatic
+        end
```

Comment:
I thought we had decided not to list `--color` in the output (see https://github.com/rspec/rspec-core/pull/2321#r77930510) since it's a deprecated option and we consider the `--help` output to be "valuable real estate" that we don't want taken up by deprecated options?

Any reason it can't be removed from the output?  (Of course, the option itself should still be honored, regardless of whether or not we display it in `--help`).